src/index.js

```diff
@@ -68,6 +68,30 @@ export default function wire(contextNamespace, mapToProps={}, mapModelToProps=no
 
 			this.mapping = mapModelToProps(get(context, contextNamespace), props);
 
+			/** Props passed to your wrapped component.
+			 *	@name props
```

Comment:
Should we say something like "in addition to the props specified in `mapToProps` and `mapModelToProps`...